lib/proteus/kit.rb

```diff
@@ -16,7 +16,9 @@ def new(kit_name, repo_name = nil)
 
       if system "git ls-remote #{url(kit_name)} #{repo_name} > /dev/null 2>&1"
         puts "Starting a new proteus-#{kit_name} project in #{repo_name}"
-        system "git clone #{url(kit_name)}#{' ' + repo_name}"
+        system "git clone #{url(kit_name)}#{' ' + repo_name} && "\
```

Comment:
This gave me an error with ` && "\` at the end of the file, but you can also escale new lines like so:
```
system "git clone #{url(kit_name)}#{' ' + repo_name} && \
   rm -rf #{repo_name}/.git && \
   git init #{repo_name}"
```